mdflynn/animetrics
# Feature/clean data

## Is this a fix or a feature?
This is a feature

## What is the change?
- Creates utilities file and moves cleaning functions to the file
- Formats data in all files

## What does it fix?
No fix this PR

## Where should the reviewer start?
The entire utilities.js files

## How should this be tested?
Run`npm start` in your terminal and navigate to localhost:3000 to see current site functionality
Run `npm test` to see a passing test suite.

## Relevant Issues
#16 #29
